styleguidist.js

```diff
@@ -0,0 +1,14 @@
+const { join } = require("path");
```

Comment:
You probably don't need this in a separate file and can be inlined into the function format of the neutrinorc.